halyard-deploy/src/main/resources/debian/install.sh

```diff
@@ -40,74 +41,35 @@ else
 fi
 
 if [ "$DISTRO" = "Ubuntu" ]; then
-  if [ "${DISTRIB_RELEASE%%.*}" -lt "14" ]; then
+  if [ "${DISTRIB_RELEASE%%.*}" -lt "18" ]; then
     echo_err "Not a supported version of Ubuntu"
-    echo_err "Version is $DISTRIB_RELEASE we require 14.04 or greater."
+    echo_err "Version is $DISTRIB_RELEASE we require 18.04 or greater."
     exit 1
   fi
 else
   echo_err "Not a supported operating system: " $DISTRO
-  echo_err "It's recommended you use Ubuntu 14.04 or greater."
+  echo_err "It's recommended you use Ubuntu 18.04 or greater."
   echo_err ""
   echo_err "Please file an issue against https://github.com/spinnaker/spinnaker/issues"
   echo_err "if you'd like to see support for your OS and version"
   exit 1
 fi
 
-function add_redis_apt_repository() {
-  # Only Ubuntu prior to 18.04 LTS requires this PPA
-  if [ "${DISTRIB_RELEASE%%.*}" -lt "18" ]; then
-    echo "Adding Redis PPA repository for Ubuntu version less than 18.04"
-    add-apt-repository -y ppa:chris-lea/redis-server
-  fi
-}
-
 function add_spinnaker_apt_repository() {
-  echo "Adding Spinnaker apt repository"
-  REPOSITORY_HOST=$(echo $REPOSITORY_URL | cut -d/ -f3)
-  echo "deb [arch=all] $REPOSITORY_URL apt main" | tee /etc/apt/sources.list.d/spinnaker.list > /dev/null
-}
-
-function add_java_apt_repository() {
-  # Only Ubuntu prior to 18.04 LTS requires this PPA
-  if [ "${DISTRIB_RELEASE%%.*}" -lt "18" ]; then
-    echo "Adding Java PPA repository for Ubuntu version less than 18.04"
-    add-apt-repository -y ppa:openjdk-r/ppa
-  fi
-}
-
-function install_java() {
```

Comment:
No longer needed because Java would have already have had to be installed in order to get Halyard installed, so if someone is able to run `hal deploy apply`, they would already have Java installed.